modules/issue_tracker/php/uploadhelper.class.inc

```diff
@@ -0,0 +1,254 @@
+<?php declare(strict_types=1);
+/**
+ * POST request method.
+ *
+ * Used for uploading attachment file for Issue Tracker.
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\issue_tracker;
+
+/**
+ * The UploadHelper Class.
+ *
+ * This class provides the code to process uploading
+ * a user file from the issue_tracker for saving attachments.
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class UploadHelper
+{
+    /**
+     * The file uploaded from the user.
+     */
+    private $_fileToUpload;
+
+    /**
+     * The bytes of the file.
+     */
+    private $_bytes;
+
+    /**
+     * The user uploading the file.
+     */
+    private $_user;
+
+    /**
+     * The 'post message' array content.
+     */
+    private $_values;
+
+    /**
+     * Setup uploading configurations.
+     *
+     * @param object $user   The user uploading.
+     * @param array  $files  The $_FILES array content.
+     * @param array  $values The post message array content.
+     *
+     * @return array
```

Comment:
Should describe the format and purpose of this array

Reply:
@johnsaigle updated